docker/build-rpm-in-docker.sh

```diff
@@ -1,6 +1,48 @@
 #!/bin/sh
-cd $(dirname $0)
-mkdir -p build-dir-$1
-cp ../* build-dir-$1
-cp -r ../dist build-dir-$1/dist
-sudo docker run -i -t -v ${PWD}/build-dir-$1:/build/build-dir-$1 erlang-rpm-build-$1
+#
+#
+#
+centos_version="$1"
+build_dir="build-dir-$centos_version"
+packages=( "Makefile" "*.patch" "erlang.spec" "Erlang_ASL2_LICENSE.txt" )
```

Comment:
Hello @Gsantomaggio . sorry i am ever have been using "bash -x" along my life. sometimes i forget the existence of scripts wrote for sh commands. there is problems if we substitute sh for bash ? i know the diference between they, advantages and utilization . what do you think about ?